Foundation.json

```diff
@@ -1,20 +1,6 @@
 {
-    "driver": {
-        "typename": "bibliopixel.drivers.SimPixel.DriverSimPixel",
-        "num": 12
-    },
-
-    "layout": {
```

Comment:
In a script, it can guess the layout from the animation type.  The only reason to specify it is if you have non-standard settings.